scripts/conditionalBuildBabelPlugin.mjs

```diff
@@ -0,0 +1,29 @@
+export default (excludedModules, t) => {
+  return {
+    visitor: {
+      ImportDeclaration(path) {
+        // Search for the comments that start with the `@skipwhen` directive.
+        const skipWhenComments =
+          path.node.leadingComments?.filter((c) => {
+            return c.value.trim().startsWith("@skipwhen");
+          }) || [];
+
+        if (skipWhenComments.length > 0) {
+          const [_, webSDKModuleName, value] = skipWhenComments[0].value.match(
+            "ENV.(.*) === (false|true)"
+          );
+
+          if (excludedModules[webSDKModuleName] === value) {
+            const variableName = path.node.specifiers[0].local.name;
+
+            path.replaceWith(
+              t.variableDeclaration("var", [
```

Comment:
Petty comment, but, instead of replacing `import createPersonalization from "../componentsPersonalization` with `var createPersonalization`, can we replace it with `const createPersonalization = null;` to keep it stylistically consistent with the rest of the code?